release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -220,6 +237,13 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Language Server
+
+- Introduced new `Extract to local variable` code action
+- Introduced new `Extract to function` code action
+- Introduced `loadProject()` API to the workspace manager
```

Comment:
Introduced `loadProject()` API to the workspace manager to open up a project programmatically.